webapp/components/Group/GroupMember.vue

```diff
@@ -53,30 +53,33 @@
         </ds-chip>
       </template>
       <template #edit="scope">
-        <ds-button v-if="scope.row.myRoleInGroup !== 'owner'" size="small" primary disabled>
-          <!-- TODO: implement removal of group members -->
-          <!--           :disabled="scope.row.myRoleInGroup === 'owner'"
-          -->
+        <ds-button
+          v-if="scope.row.myRoleInGroup !== 'owner'"
+          size="small"
+          primary
+          @click="
+            isOpen = true
+            userId = scope.row.id
+          "
+        >
```

Comment:
Would be cool if you can use the `base-button`, because then the colors of Yunite would fit from scratch.
Sorry that I have overlooked this at the first review.

Somehow the button is now always disabled:

<img width="1644" alt="Bildschirmfoto 2023-03-07 um 19 50 22" src="https://user-images.githubusercontent.com/25344101/223524252-36ed8eb7-e967-47e3-8293-ad47702e5faa.png">

<img width="1644" alt="Bildschirmfoto 2023-03-07 um 19 55 50" src="https://user-images.githubusercontent.com/25344101/223524432-b97ef1bf-a77a-47c1-b03f-d97c92efbaa5.png">

Reply:
@Tirokk I changed the button. I cannot reproduce your problem

Reply:
![image](https://user-images.githubusercontent.com/15882241/223558384-e962dee7-a9b5-4f73-aeff-5610f811cb4d.png)

Reply:
I idiot was on the wrong branch yet ... 🙈